app/controllers/api/companies_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::CompaniesController < Api::BaseController
+  # This is protected by API token
+  def index
+    @companies = @current_person.companies
```

Comment:
from where this instance variable is coming ?
@current_person

remove this magic.

change to current_person as method